test/jdk/sun/security/util/math/TestIntegerModuloP.java

```diff
@@ -22,24 +22,85 @@
  * @build BigIntegerModuloP
  * @run main TestIntegerModuloP sun.security.util.math.intpoly.IntegerPolynomial25519 32 0
+ */
+
+ /*
+ * @test id=IntegerPolynomial448
+ * @modules java.base/sun.security.util java.base/sun.security.util.math java.base/sun.security.util.math.intpoly
+ * @build BigIntegerModuloP
  * @run main TestIntegerModuloP sun.security.util.math.intpoly.IntegerPolynomial448 56 1
+ */
+
+ /*
+ * @test id=IntegerPolynomial1305
+ * @modules java.base/sun.security.util java.base/sun.security.util.math java.base/sun.security.util.math.intpoly
+ * @build BigIntegerModuloP
  * @run main TestIntegerModuloP sun.security.util.math.intpoly.IntegerPolynomial1305 16 2
+ */
+
+ /*
+ * @test id=IntegerPolynomialP256
+ * @modules java.base/sun.security.util java.base/sun.security.util.math java.base/sun.security.util.math.intpoly
+ * @build BigIntegerModuloP
  * @run main TestIntegerModuloP sun.security.util.math.intpoly.IntegerPolynomialP256 32 5
+ */
+
+ /*
+ * @test id=IntegerPolynomialP384
+ * @modules java.base/sun.security.util java.base/sun.security.util.math java.base/sun.security.util.math.intpoly
+ * @build BigIntegerModuloP
  * @run main TestIntegerModuloP sun.security.util.math.intpoly.IntegerPolynomialP384 48 6
+ */
+
+ /*
+ * @test id=IntegerPolynomialP521
+ * @modules java.base/sun.security.util java.base/sun.security.util.math java.base/sun.security.util.math.intpoly
+ * @build BigIntegerModuloP
  * @run main TestIntegerModuloP sun.security.util.math.intpoly.IntegerPolynomialP521 66 7
+ */
+
+ /*
+ * @test id=P256OrderField
+ * @modules java.base/sun.security.util java.base/sun.security.util.math java.base/sun.security.util.math.intpoly
+ * @build BigIntegerModuloP
  * @run main TestIntegerModuloP sun.security.util.math.intpoly.P256OrderField 32 8
+ */
+
+ /*
+ * @test id=P384OrderField
+ * @modules java.base/sun.security.util java.base/sun.security.util.math java.base/sun.security.util.math.intpoly
+ * @build BigIntegerModuloP
  * @run main TestIntegerModuloP sun.security.util.math.intpoly.P384OrderField 48 9
+ */
+
+ /*
+ * @test id=P521OrderField
+ * @modules java.base/sun.security.util java.base/sun.security.util.math java.base/sun.security.util.math.intpoly
+ * @build BigIntegerModuloP
  * @run main TestIntegerModuloP sun.security.util.math.intpoly.P521OrderField 66 10
+ */
+
+ /*
+ * @test id=Curve25519OrderField
+ * @modules java.base/sun.security.util java.base/sun.security.util.math java.base/sun.security.util.math.intpoly
+ * @build BigIntegerModuloP
  * @run main TestIntegerModuloP sun.security.util.math.intpoly.Curve25519OrderField 32 11
+ */
+
+ /*
+ * @test id=Curve448OrderField
+ * @modules java.base/sun.security.util java.base/sun.security.util.math java.base/sun.security.util.math.intpoly
+ * @build BigIntegerModuloP
  * @run main TestIntegerModuloP sun.security.util.math.intpoly.Curve448OrderField 56 12
  */
 
+
```

Comment:
Excess new line.

Reply:
Updated with Copyright and removed the excess line. Please see latest commit.